packages/identifier/src/banned.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * This file is meant for 100% no-go, banned words that should have fuzzier matching in order to catch more usernames (ie,
+ * explicitly use regexes before and after each of these strings in order to catch all instances of these words being used.)
+ */
+
+// naive, incomplete list of slurs and unacceptable words
+const slurs = [
+  'bluegum',
+  'chink',
+  'chinks',
+  'coolie',
+  'coolies',
+  'coon',
+  'coons',
```

Comment:
With the change of dealing with this via regex, the word "raccoon" will be blocked as well. I don't disagree with the basic premise of the PR, but there should be a way to allow innocuous words that otherwise might get caught in the slur filter.

Reply:
Agreed - though, seems the bluesky team’s got their own solution that looks like it’ll solve that issue https://github.com/bluesky-social/atproto/pull/1336